lib/structure/structureLoader.js

```diff
@@ -550,8 +550,8 @@ function reorder (obj, validatorExtension) {
   return ordered;
 }
 
-function load (dir, validatorExtension, callback) {
-  analyze(dir, function (err, dividedByTypes, warns) {
+function load (dir, validatorExtension, useLoaderExtensions, callback) {
```

Comment:
I use this function directly in my system and the signature changed (with the additional 'useLoaderExtensions' parameter).